tests/openapi/run.sh

```diff
@@ -15,6 +15,23 @@ function prepare_database() {
 	run_sql 'CREATE DATABASE openapi;' $MYSQL_PORT2 $MYSQL_PASSWORD2
 }
 
+function init_noshard_data() {
```

Comment:
what's the difference between `init_shard_data`?

and you can use `run_sql_source1`

Reply:
😂  init_shard_data: source-1 and source-2 have same table name  but `init_no_shard_data` does not